specification/components/schemas/headers/response/ContentDispositionAdviceAndGuidanceAttachment.yaml

```diff
@@ -0,0 +1,6 @@
+description: | 
```

Comment:
Should this component have a `name` attribute?

Reply:
This is a copy of the standard content disposition attachment but with the example updated to an A&G